tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -498,3 +498,21 @@ Feature: antivirus
       | Virus found in readme.md. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
     And for user "Brian" the content of the file ".space/readme.md" of the space "new-space" should be "Here you can add a description for this Space."
     And for user "Alice" the content of the file ".space/readme.md" of the space "new-space" should be "Here you can add a description for this Space."
+
+
+  Scenario: try to overwrite a file with virus content in project space share
```

Comment:
i think rather than using try it is better to define what action is performing and what result will be occur

Reply:
```suggestion
  Scenario: overwriting a file with file containing virus in project space share
```
how about this scenario

Reply:
looks good

Reply:
I would argue to use `try to overwrite a file with virus content in project space share` because in the scenario what we're doing is trying (we already expect it to not work) while `overwriting a file with file containing virus in project space share` gives a false impression that the scenario is expected to work? 

That just my opinion though 

Also, I think it should be `overwriting a file with virus content in project space share` not `file with file`

Reply:
I was confused because of HTTP status code 204. I thought it is a successful scenario. Even its negative test scenario maybe we can say something like `can't overwrite file `.